gateway/src/apicast/version.lua

```diff
@@ -1 +1 @@
-return "3.6.0"
+return "3.7.0"
```

Comment:
We should use `3.7.0-beta1`.
We switch to `3.7.0` in the CR because CRs can become final.